src/applications/terms-of-use/containers/TermsOfUse.jsx

```diff
@@ -17,7 +17,7 @@ import {
 import { touStyles, errorMessages } from '../constants';
 import touData from '../touData';
 
-const touUpdatedDate = `September 2023`;
+const touUpdatedDate = `March 2024`;
```

Comment:
❓ Should this reflect the date the ToU changes were proposed ("March 2024") or the date they actually land ("April 2024")?